src/System.Private.CoreLib/shared/System/Char.cs

```diff
@@ -904,7 +904,14 @@ public static bool IsSurrogatePair(string s, int index)
 
         public static bool IsSurrogatePair(char highSurrogate, char lowSurrogate)
         {
-            return IsHighSurrogate(highSurrogate) && IsLowSurrogate(lowSurrogate);
+            // Since both the high and low surrogate ranges are exactly 0x400 elements
```

Comment:
Worth asserting this?

Reply:
I don't think so. I've not really seen assertions of const values throughout the framework since it seems redundant. In this case, the unit tests would catch problems if we ever accidentally change the value of the const.

Reply:
> since it seems redundant

It would be about validating the relationship between them, not validating the value itself.

If you don't think it's useful, ok.